index.html

```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
       </div>
     </section>
 
+    <section class="work">
+      <div class="card">
+        <img src="assets/images/mobile/portfolio1.png" alt="project1">
+        <div class="card-text">
+          <div class="card-header">
+            <h3>Tonic</h3>
+            <div class="frame">
+              <ul>
+                <li class="client"><h4>CANOPY</h4></li>
+                <li class="role"><i class="fa fa-circle"></i><h4>Back End Dev</h4></li>
+                <li class="year"><i class="fa fa-circle"></i><h4>2015</h4></li>
+              </ul>
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="tag">
+                <ul>
+                  <li class="tag1"><h3>HTML</h3></li>
+                  <li class="tag2"><h3>CSS</h3></li>
+                  <li class="tag3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
```

Comment:
Please correct your design issue as described below. Do this for other parts of your code as well that has this issue.
- [ ] Your `javascript` tag falls out its border
- [ ] Please add more `padding-right` to your **`See Project`** links to look more like the template

![emma-js-btns](https://user-images.githubusercontent.com/36542986/134599029-e183794e-abdf-4196-b5dc-e4cec36cf5f8.jpg)